lib/pure/asynchttpserver.nim

```diff
@@ -97,6 +100,18 @@ proc respond*(req: Request, code: HttpCode, content: string,
 
   if headers != nil:
     msg.addHeaders(headers)
+  msg.add("Content-Length: ")
```

Comment:
Now that I look at it, there should probably be a check to ensure ``headers`` don't contain a ``Content-Length`` header.